Profesja/Include/Profesja.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+#include <iostream>
+#include <vector>
+#include <string>
+
+struct Player
+{
+  unsigned int currentHp = 20;
+  unsigned int power = 1;
+  std::vector <std::string> handdeck = {"karta1", "karta2", "karta3"};

```

Comment:
1. `<str::string>` dorównać do lewej
2. Nie tworzy się tak decku defaultowego
3. Deck powinno być osobną klasą, Player powinien trzymać trzy wskaźniki na obiekty tej klasy dla ręki, odrzuconego, dobieranego stosu.
4. Deck powinien implementować wszystkie rzeczy na karcie (Card to powinna być kolejna klasa, ale na razie wystarczy `using Card = std::string`)

Reply:
@mnurzyns Ja tutaj zastosowałem podstawowy deck, żebym mógł napisać już funkcje. Prawdziwy Deck ma się znajdować w Playerze, i jak dobrze wiem jest to robota grupy Points.